src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/shipping_method/create.yaml

```diff
@@ -44,12 +44,18 @@ sylius_twig_hooks:
             enabled:
                 template: '@SyliusAdmin/shipping_method/form/general/enabled.html.twig'
                 priority: 200
+            min_delivery_time_days:
+                template: '@SyliusAdmin/shipping_method/form/general/min_delivery_time_days.html.twig'
+                priority: 100
+            max_delivery_time_days:
+                template: '@SyliusAdmin/shipping_method/form/general/max_delivery_time_days.html.twig'
+                priority: 0
             zone:
                 template: '@SyliusAdmin/shipping_method/form/general/zone.html.twig'
-                priority: 100
+                priority: -100
```

Comment:
Do not change the priorities for existing hookables, just add new ones in between